05-onchain.md

```diff
@@ -163,80 +163,86 @@ In the case of a mutual close, a node need not do anything else, as it has
 already agreed to the output, which is sent to its specified `scriptpubkey` (see
 [BOLT #2: Closing initiation: `shutdown`](02-peer-protocol.md#closing-initiation-shutdown)).
 
-# Unilateral Close Handling: Local Own Commitment Transaction
+# Unilateral Close Handling: Local Commitment Transaction
 
-This is the first of two cases involving unilateral closes: in this case, a
-node sees *its own commitment transaction*, which *resolves* the funding
+This is the first of two cases involving unilateral closes. In this case, a
+node discovers its *local commitment transaction*, which *resolves* the funding
 transaction output.
 
-However, a node cannot claim funds from the outputs of its own unilateral close
-until the `OP_CHECKSEQUENCEVERIFY` delay has passed (as specified by
-the other node's `to_self_delay` field). Where relevant, this is noted below.
+However, a node cannot claim funds from the outputs of a unilateral close that
+it initiated, until the `OP_CHECKSEQUENCEVERIFY` delay has passed (as specified
+by the remote node's `to_self_delay` field). Where relevant, this situation is
+noted below.
 
 ## Requirements
 
-When a node sees *its own* commitment transaction:
-
-1. `to_local` output: A node SHOULD spend this output to a convenient address.
-   A node MUST wait until the `OP_CHECKSEQUENCEVERIFY` delay has passed (as specified by the remote
-   node's `to_self_delay` field) before spending the output. If the
-   output is spent (as recommended), the output is *resolved* by the spending
-   transaction, otherwise it is considered *resolved* by the commitment transaction itself.
-2. `to_remote` output: No action required, this output is considered *resolved*
-   by the commitment transaction itself.
-3. HTLCs offered by this node: See "HTLC Output Handling: Local Commitment, Local Offers" below.
-4. HTLCs offered by the remote node: See "HTLC Output Handling: Local Commitment, Remote Offers" below.
+A node:
+  - upon discovering its *local commitment transaction*:
+    - SHOULD spend the `to_local` output to a convenient address.
+    - MUST wait until the `OP_CHECKSEQUENCEVERIFY` delay has passed (as
+    specified by the remote node's `to_self_delay` field) before spending the
+    output.
+      - Note: if the output is spent (as recommended), the output is *resolved*
+      by the spending transaction, otherwise it is considered *resolved* by the
+      commitment transaction itself.
+    - MAY ignore the `to_remote` output.
+      - Note: No action is required by the local node, as `to_remote` is
+      considered *resolved* by the commitment transaction itself.
+    - [FIXME: SHOULD|MAY|MUST?] handle HTLCs offered by itself as specified in
+    "HTLC Output Handling: Local Commitment, Local Offers" below.
+    - [FIXME: SHOULD|MAY|MUST?] handle HTLCs offered by the remote node as
```

Comment:
MUST.  Otherwise it weakens any MUST on those sections...

Reply:
done, makes sense